lib/Doctrine/ORM/Cache/Exception/InvalidResultCacheDriver.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Cache\Exception;
+
+final class InvalidResultCacheDriver extends \Exception implements CacheException
+{
+    public static function create() : self
```

Comment:
Generic `create()` method doesn't say much, could we have them express what they really do? I.e. `notImplementingCacheInterface()`?

Reply:
Won't that be a bit long?